src/library_egl.js

```diff
@@ -136,17 +212,17 @@ var LibraryEGL = {
       {{{ makeSetValue('value', '0', '8' /* 8 bits for red channel. */, 'i32') }}};
       return 1;
     case 0x3025: // EGL_DEPTH_SIZE
-      {{{ makeSetValue('value', '0', '24' /* 24 bits for depth buffer. TODO: This is hardcoded, add support for this! */, 'i32') }}};
+      {{{ makeSetValue('value', '0', 'EGL.configAttribs[config-1].depth' /* 16 bits for depth buffer. */, 'i32') }}};
```

Comment:
https://www.khronos.org/registry/webgl/specs/latest/1.0/#5.2.1 - you're right in that desktop browsers do generally go for 24 bits. The spec only promises "at least 16 bits" as that's lower limit GLESv2 guarantees to provide, which definitely is the case on mobile chips. (leaving aside that Emscripten apps run like a sack of wet hammers on mobile)